include/ignition/physics/Geometry.hh

```diff
@@ -56,6 +56,10 @@ namespace ignition
     using AngularVector = Vector<Scalar, (Dim*(Dim-1))/2>;
     IGN_PHYSICS_MAKE_ALL_TYPE_COMBOS(AngularVector)
 
+    template <typename Scalar, std::size_t Dim>
+    using Wrench = Vector<Scalar, Dim + (Dim*(Dim-1))/2>;
+    IGN_PHYSICS_MAKE_ALL_TYPE_COMBOS(Wrench)
```

Comment:
is there an advantage to using a single vector over `std::pair<LinearVector, AngularVector>` or a struct? I would have trouble remembering if the linear or angular part of the wrench came first.

Reply:
Switched to a struct in https://github.com/ignitionrobotics/ign-physics/pull/283/commits/1bd5c0091da6f63550d923b327c61046a01782eb